build/scripts/bootstrap.sh

```diff
@@ -68,21 +68,25 @@ function installBuildDependencies
 
         apt-get install -y make build-essential g++ gdb gdbserver gcc git wget
         apt-get install -y python3 ninja-build
-        apt-get install -y cmake libmsgsl-dev
         apt-get install -y libboost-program-options-dev
         apt-get install -y libproxy-dev libssl-dev uuid-dev libcurl4-openssl-dev
+        apt-get install -y cmake libmsgsl-dev
+        apt-get install -y file
```

Comment:
>apt-get install -y file [](http://example.com/codeflow?start=8&length=23)

Add an inline comment that says this is required for CPack and is not installed by default in Ubuntu 22.04.